src/components/ContractForm/ContractForm.tsx

```diff
@@ -45,19 +49,22 @@ export const ContractForm: Component<Props> = ({ state, actions, key }) => {
             min={startDate.value}
             onInputChange={updateContractFormValue(key, formActions)}
           />
+          {endDate.errors == null && <FormHint label={String.fromCharCode(160)} />}
+          {endDate.errors != null && endDate.errors.negativeDuration && <FormHint theme="danger" label="Contract has negative duration" />}
         </FormField>
       </div>
       <div className="contract-form__column">
-        <FormField labelText="Pensum" required={true} hint="0-100">
+        <FormField labelText="Pensum" required={true}>
           <FormInput
             name={pensumPercentage.name}
             value={pensumPercentage.value}
             type="number"
             suffix="fas fa-percent"
-            min={0}
-            max={100}
+            min={CONTRACT_PENSUM_VALUE_MIN}
```

Comment:
you might want to add errors here too.

Reply:
We eliminated the textLengthValidator in favor of HTML input limits.
Spinner has limits set, so a validator is obsolete.